rwengine/src/script/modules/GameModule.cpp

```diff
@@ -803,7 +803,7 @@ void game_create_crusher_crane(const ScriptArguments& args)
 	// NOTE: This instruction also creates or controls a garage
 	/// @todo find out if this creates a garage or if it just controls garage[0]
 	args.getWorld()->state->garages.push_back({
-		min, max, garageType
+		0, min, max, garageType
```

Comment:
It's used to determine the number to write to the script output when assigning to a ScriptGarage.